ui-v2/app/components/consul-service-instance-list/index.hbs

```diff
@@ -6,73 +6,118 @@
       </a>
     </BlockSlot>
     <BlockSlot @name="details">
-      <ul>
-        <ConsulExternalSource @item={{item.Service}} as |ExternalSource|>
-          <li>
-            <ExternalSource />
-          </li>
-        </ConsulExternalSource>
+      <ConsulExternalSource @item={{item.Service}} as |ExternalSource|>
+        <ExternalSource />
+      </ConsulExternalSource>
```

Comment:
If you don't need to wrap anymore this it can just be a `<ConsulExternalSource @item={{item.Service}}/>` now.